rdflib/namespace.py

```diff
@@ -95,11 +95,6 @@ def __new__(cls, value):
             rt = unicode.__new__(cls, value, 'utf-8')
         return rt
 
-
-    @property
-    def title(self):
-        return URIRef(self + 'title')
-
```

Comment:
I think it was to make this:

    >>> from rdflib.namespace import DCTERMS
    >>> DCTERMS.title

result in:

    rdflib.term.URIRef(u'http://purl.org/dc/terms/title')

instead of:

    u'Http://Purl.Org/Dc/Terms/'